src/averageAges.js

```diff
@@ -15,12 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const men = century
+    ? people.filter((gender) => gender.sex === 'm'
+      && Math.ceil(gender.died / 100) === century)
+    : people.filter((gender) => gender.sex === 'm');
```

Comment:
You filter people in both cases. You can just move the ternary operator inside callback of one filter method to make your code cleaner. And it's not a gender, but person